package.json

```diff
@@ -13,7 +13,7 @@
     "autoprefixer": "^10.4.20",
     "babel-loader": "^9.2.1",
     "caniuse-lite": "^1.0.30001667",
-    "canvas": "^2.11.2",
+    "canvas": "3.0.0-rc2",
```

Comment:
Here, and in the Gulpfile, is there a reason why we can't use `^3.0.0-rc2`? That would more explicitly indicate that we also accept new versions when they are released, which would also be good to have considering that we (for now at least) have to rely on a release candidate version.

Reply:
That seems to work just fine, thank you.